src/Tasks/Build/Template.php

```diff
@@ -67,11 +67,7 @@ public function run()
         $files = $this->copyTarget($this->source, $this->target);
 
         // Build media (relative path)
-        $media = $this->buildMedia("media/" . $this->templateName, $this->templateName);
-        $media->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
-            ->run();
-
-        $this->addFiles('media', $media->getResultFiles());
+        $this->buildMediaFiles('tpl_' . $this->templateName);
```

Comment:
```suggestion
        $this->buildMediaFiles($this->templateName);
```
templates don't use the tpl_ prefix

Reply:
I'm just handing in the name of the extension and the prefix lets the system discover which type it is. In this case, the tpl_ is necessary.